[Service Bus] Porting 14692 to the patch version (#14720)
Azure/azure-sdk-for-js: sdk/servicebus/service-bus/CHANGELOG.md

# Release History

## 7.0.5 (2021-04-06)

### Bug fixes

> - Some of the queue properties such as "forwardTo" and "autoDeleteOnIdle" were not being set as requested through the `ServiceBusAdministrationClient.createQueue` method because of a bug w.r.t the ordering of XML properties. The issue has been fixed in [#14692](https://github.com/Azure/azure-sdk-for-js/pull/14692).

Reviewer: Nit pick, can we spell out "with regards to"? Some may not be familiar with the acronym:
```suggestion
- Some of the queue properties such as "forwardTo" and "autoDeleteOnIdle" were not being set as requested through the `ServiceBusAdministrationClient.createQueue` method because of a bug with regards to the ordering of XML properties. The issue has been fixed in [#14692](https://github.com/Azure/azure-sdk-for-js/pull/14692).
```
